ViewModels/PlantsViewModel.swift

```diff
@@ -32,6 +31,9 @@ class PlantsViewModel: ObservableObject {
 	
 	@Published var plants: [PlantEntity] = []
 	
+	@Published var singlePlant: PlantEntity?
+	
+	// TODO: change relationship between viewmodel and datacontroller, and injection of the latter
```

Comment:
I'm not sure the NSManagedObjectContext should be a property on VM, but rather on DataController (i.e., repository class). Since the latter is has a static instance, it will always be available anyway. I'm making an issue about this.